### PR TITLE
Authentication sign up

### PR DESCRIPTION
**Pull request by ritikaawasthy**

### CODE REVIEW

**Review by Debarshi95 — APPROVED** [review]
All good. Just check the suggestion in previous PR.

File: css/styles.css [hunk 1]
```diff
@@ -1,10 +1,57 @@
...
+  z-index: 7;
+  max-width: 20rem;
+  height: 30rem;
+  border: 2px solid var(--primary-light);
+  box-shadow: 1px 1px 4px 4px var(--primary-light);
+  color: var(--primary);
+  background-color: var(--primary-light);
+}
+
+#side-menu:li) {
```
Comment (swethaogeti) [inline css/styles.css:38]: is this working in the line li) just asking

File: templates/home.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                <i class="fas fa-angles-left card-icon left f-l " id="slid-t-btn-rev"></i>
+                <p>1920</p>
+              </div>
+            </div>
+        </article>
+      </div>
+        <div class="container fl-start">
+        <article class="grid-two-layout-left">
+            <div class="card w-xl">
+              <img src="/media/home/right-top-1.webp" id="slid-t-img-right"  alt="" class="background">
```
Comment (swethaogeti) [inline templates/home.html:80]: add alt

File: templates/signup.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+</head>
+
+<body id="login-bg">
+  <header>
+    <nav class="nav">
+      <div class="start">
+        <h2 class="theme-heading primary-col">Thrify with me</h2>
+      </div>
+      <ul class="list list-inline f-m">
+        <li class=''>
```
Comment (swethaogeti) [inline templates/signup.html:20]: undefined class in line 20

**Review by sanjitsarkar — APPROVED** [review]
Looks good

File: templates/login.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    <section class="grid-three-layout-center">
+      <article class="card card-shadow l">
+        <div class="background">
+          <img src="/media/auth/login.jpg" alt="">
+        </div>
+        <div class="card-content stacked">
+          <h2 class="center primary-col f-xl fw-li">Login</h2>
+          <!-- <div class="card-content stacked"> -->
+          <div class="input-container ">
+            <input placeholder=" " class="input w-m" type="email">
```
Comment (swethaogeti) [inline templates/login.html:44]: you can remove placeholder if not used

File: templates/signup.html [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+            </div>
+            <div class="input-container ">
+              <input placeholder=" " class="input w-m" type="password">
+              <label class="input-label f-s fw-li">Re-Password</label>
+            </div>
+          </div>
+          <a href="#" class="btn f-s primary center">Login</a>
+          <div class="card-content inline end primary-col ">
+            <i class="fa-brands fa-google f-l"></i>
+            <i class="fa-brands fa-facebook f-l"></i>
```
Comment (swethaogeti) [inline templates/signup.html:69]: rest is good ! clean code ! GTM

**Review by swethaogeti — APPROVED** [review]
clean code! u can merge

File: css/styles.css (hunk 1 above)
Comment (ritikaawasthy) [reply css/styles.css:38]: thanks for pointing this out

**Pull request merged**